app/onboarding/page.tsx

```diff
@@ -599,8 +599,26 @@ export default function OnboardingPage() {
             </Flex>
             <Separator mt={4} />
           </Box>
-
-          <Flex alignItems="center" gap={3} mt={8}>
+          <Box
+            mt={4}
+            p={4}
+            borderWidth="1px"
+            borderRadius="md"
+            bg="lime.100"
+            borderColor="lime.400"
```

Comment:
```suggestion
            bg="secondary.subtle"
            borderColor="secondary.solid"
```
Uses semantic secondary color